_posts/2019-01-24-how-to-keep-safe-as-a-client.md

```diff
@@ -0,0 +1,129 @@
+---
+layout: post
+title:  "how to keep safe as a client"
+date:   2019-01-27 22:00:00 +0800
+categories: blog
+author: Joe Zhang
+---
+
+[Artical Link](https://mp.weixin.qq.com/s?__biz=MzAxODcyNjEzNQ==&mid=2247486481&idx=1&sn=87aee20e301d87030be2636cd0a124b7&chksm=9bd0a189aca7289f0a5e8a91907d21e32bd367341251c713e76c2fd97f6f64c06379ad7c4f93&scene=21#wechat_redirect): 
+
+Previously I worked on a project which use many third party services, These third party services exposed restful APIs and my project will act as a client.
+In such background, If some services were broken or having high latency, It will affect our system. what's more, It will block user request and may cause the whole system broken down.
```

Comment:
Ok, the general rule is that a period (.) denotes the end of sentence, so you capitalize the next word that starts the next sentence. But a comma (,), split a sentence into smaller parts, you don't need to capitalize the word after commas.

Reply:
thanks jack, I just fixed grammar issues and capitalize issues.